script/prerender/createRedirects.ts

```diff
@@ -0,0 +1,43 @@
+import fs from 'fs';
+import path from 'path';
+import { Redirects } from '../../data/redirects/types';
+import { content } from '../../src/app/content/routes';
+import { makeUnifiedBookLoader } from '../../src/app/content/utils';
+import { findArchiveTreeNodeById } from '../../src/app/content/utils/archiveTreeUtils';
+import { AppServices } from '../../src/app/types';
+import { writeAssetFile } from './fileUtils';
+
+const redirectsPath = path.resolve(__dirname, '../../data/redirects/');
+
+const createRedirects = async(archiveLoader: AppServices['archiveLoader'], osWebLoader: AppServices['osWebLoader']) => {
+  const bookLoader = makeUnifiedBookLoader(archiveLoader, osWebLoader);
+
+  const books = fs.readdirSync(redirectsPath).filter((name) => name.match('.json'));
+
+  const redirects: Array<{ from: string, to: string }> = [];
+
+  for (const bookName of books) {
+    const bookRedirects: Redirects = require(redirectsPath + '/' + bookName);
```

Comment:
same thing about the mixed require/import, we should be using import consistently or we'll have compatibility issues